scripts/pre_push_hook.py

```diff
@@ -295,21 +295,20 @@ def _install_hook():
         print 'Copied file to .git/hooks directory'
 
 
-def does_diff_include_js_files(files_to_lint):
-    """Returns true if diff includes JS files.
+def does_diff_include_frontend_scripts(files_to_lint):
+    """Returns true if diff includes TypeScript or JavaScript files.
 
     Args:
         files_to_lint: list(str). List of files to be linted.
 
     Returns:
-        bool. Status of JS files in diff.
+        bool. Whether the diff contains changes in any frontend script.
```

Comment:
May be rephrase this to `Whether the diff includes any frontend file.`

Reply:
Ditto (html can also be the frontend files.)